packages/blocks/src/api/validation/index.js

```diff
@@ -407,9 +407,7 @@ export const isEqualAttributesOfName = {
 			...[ actual, expected ].map( getTextPiecesSplitOnWhitespace )
 		).length;
 	},
-	style: ( actual, expected ) => {
-		return isEqual( ...[ actual, expected ].map( getStyleProperties ) );
-	},
+	style: () => true,
```

Comment:
```suggestion
         // Minor differences in style attributes should not cause blocks to be considered invalid.
         // Consider different `style` attributes to be equivalent (if not strictly equal).
         // See: https://github.com/WordPress/gutenberg/issues/37851
	style: () => true,
```

This change seems significant enough to warrant an inline comment.